### ISSUE TITLE
Inquiry About PBR Dataset Size and Resolution

### ISSUE THREAD

**Issue by ra0429** [IHe-KaiI/DressCode#24]
Hi,

Thank you for your amazing work on DressCode! I am very interested in the PBR dataset used in your research. Could you provide details on:

1.The total size of the dataset (number of samples).
2.The resolution of the textures in the dataset.

This information would be very helpful. Looking forward to your response! 
Thank you very much!

**Comment by IHe-KaiI** [comment]
Hi, 

Thanks for your interest in our project!

We train with about 2k samples on 512x512 resolution.

**Comment by ra0429** [comment]
> Hi,
> 
> Thanks for your interest in our project!
> 
> We train with about 2k samples on 512x512 resolution.

Wow, only 2K samples? That’s really impressive! I would have expected a much larger dataset for training such a model.

Could you share more details on the loss functions you used to achieve good results with such a small dataset? Did you apply any specific techniques to enhance generalization or stability during training?

**Comment by IHe-KaiI** [comment]
As discussed in Section 4.1 of the paper, we first freeze the VAE and only fine-tune the UNet for diffuse maps, and then freeze the trained UNet and fine-tune two VAEs for normal maps and roughness maps. We find this progressive training strategy facilitates a more stable and efficient process.

**Comment by ra0429** [comment]
> As discussed in Section 4.1 of the paper, we first freeze the VAE and only fine-tune the UNet for diffuse maps, and then freeze the trained UNet and fine-tune two VAEs for normal maps and roughness maps. We find this progressive training strategy facilitates a more stable and efficient process.

Thank you for the clarification!
